front/components/poke/PokeFavorites.tsx

```diff
@@ -26,6 +28,11 @@ export function PokeFavoriteButton({ title }: PokeFavoriteButtonProps) {
     toggleFavorite(createFavorite(url, title));
   }, [toggleFavorite, url, title]);
 
+  useEffect(() => {
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm not sure since when we add this rule but we should remove this linter rule, I will take a look after 🙏